runner/app/main.py

```diff
@@ -54,6 +54,9 @@ def load_pipeline(pipeline: str, model_id: str) -> any:
             from app.pipelines.segment_anything_2 import SegmentAnything2Pipeline
 
             return SegmentAnything2Pipeline(model_id)
+        case "llm-generate":
```

Comment:
@kyriediculous can we rename this to `chat-completion`? See discussion [here](https://discord.com/channels/423160867534929930/1285627017001308203/1287769210268811350).